src/main/main.js

```diff
@@ -412,24 +402,26 @@ function handleAppGPUProcessCrashed(event, killed) {
   console.log(`The GPU process has crashed (killed = ${killed})`);
 }
 
+function openDeepLink(deeplinkingUrl) {
```

Comment:
Should we not use this above in the `handleAppSecondInstance` handler?

Reply:
definitely, good catch!